api/mraa/gpio.h

```diff
@@ -220,6 +220,14 @@ int mraa_gpio_get_pin(mraa_gpio_context dev);
  */
 int mraa_gpio_get_pin_raw(mraa_gpio_context dev);
 
+/**
+ * Get the SYSFS pin path, invalid will return NULL
+ *
+ * @param dev The Gpio context
+ * @return Gpio pin sysfs path
```

Comment:
You refer to the comment line and I refer to the `@return` one. Adding it to `@return` will have it fully picked up by the doxygen, meaning proper/better documenting.